pkg/iss/iss.go

```diff
@@ -920,7 +920,7 @@ func sequenceNumbers(start t.SeqNr, step t.SeqNr, length int) []t.SeqNr {
 
 // reqStrKey takes a request reference and transforms it to a string for using as a map key.
 func reqStrKey(reqRef *requestpb.RequestRef) string {
-	return fmt.Sprintf("%d-%d.%v", reqRef.ClientId, reqRef.ReqNo, reqRef.Digest)
+	return fmt.Sprintf("%s-%d.%v", reqRef.ClientId, reqRef.ReqNo, reqRef.Digest)
```

Comment:
Let's use `%v` so we don't need to change this again in the unlikely case we use a different type for IDs in the future (or in a variant of Mir)